test/testUtil.ts

```diff
@@ -9,45 +9,94 @@ import path = require("path");
 import Q = require("q");
 
 export class TestUtil {
-
-    public static MOCK_SERVER_OPTION_NAME: string = "--mockserver";
-    public static PLATFORM_OPTION_NAME: string = "--platform";
-    public static TARGET_OPTION_NAME: string = "--target";
+    public static ANDROID_PLATFORM_OPTION_NAME: string = "--android";
+    public static IOS_PLATFORM_OPTION_NAME: string = "--ios";
     public static SHOULD_USE_WKWEBVIEW: string = "--use-wkwebview";
+    public static TEST_RUN_DIRECTORY: string = "--test-directory";
+    public static TEST_UPDATES_DIRECTORY: string = "--updates-directory";
+    public static CORE_TESTS_ONLY: string = "--core-tests";
+    public static PULL_FROM_NPM: string = "--npm";
+    
+    public static IOSServerUrl = "http://127.0.0.1:3000";
+    public static AndroidServerUrl = "http://10.0.2.2:3000";
```

Comment:
For Android, this is not always the case. In the case of Visual Studio Android Emulator the localhost loopback for the emulator is different, that's why I believe keeping it as a mocha command line parameter is a good idea.